init.rb

```diff
@@ -28,7 +28,12 @@
 end
 
 
-if Rails::VERSION::MAJOR >= 3
+if Rails::VERSION::MAJOR >= 5 and Rails::VERSION::MINOR >= 1
+  ActiveSupport::Reloader.to_prepare do
```

Comment:
We can use `Rails.configuration.to_prepare` on both Redmine 3 and 4 like #80 did.